src/controllers/sheltersController.js

```diff
@@ -14,6 +16,27 @@ const getShelters = async (req, res, next) => {
 		});
 };
 
+const postShelters = async (req, res, next) => {
+
+	//check all attrs are provided (input validation)
```

Comment:
According to the API endpoints section in Confluence, this endpoint also needs a 406 and 401. We can probably wait to implement 401 until I get the user/shelter creation stuff up and running

Reply:
Yup. I added a comment regarding those HTTP status in the Jira comment section.